### PR TITLE
adding a default export to messageTypes to fix eslint

### PR DESCRIPTION
**Pull request by julien51**
# Description

Follow on of https://github.com/unlock-protocol/unlock/pull/4668
There was also breakage in unlock-app.

# Checklist:

- [X] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [X] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [ ] My code follows the style guidelines of this project, including naming conventions
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] If my code adds or changes components, I have written corresponding stories with Storybook
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

**Comment by cnasc** [comment]
I think the recent lint upgrades broke a bunch of things.

**Comment by julien51** [comment]
dang. I a re-installing things locally to look at that.

**Comment by cnasc** [comment]
```
[breaking] enable `react/jsx-props-no-spreading`
```

https://github.com/unlock-protocol/unlock/pull/4652

Issue introduced here

**Comment by julien51** [comment]
yup. I'll update this PR with fixes.

**Comment by julien51** [comment]
So, I just pushed a large update which adds changes to a lot of files. There is no change in logic, just stylistic changes.

File: scripts/monorepo.sh [hunk 1]
```diff
@@ -7,8 +7,10 @@ BRANCH=$3
 # This script is highly specific to circleci and will "succeed" early when the
 # current git changes are not relevant to the folder being tested (and not on master)
 
+SHARED_CONFIG_FILES=(package.json .eslintrc.js .prettierrc)
```
Comment (julien51) [inline scripts/monorepo.sh]: This is really the only important change here: it makes sure that if we change any of these 3 files, tests (including linting) will be run again in the subdirectory.

This should reduce the likeliness of this mess.

**Comment by julien51** [comment]
There is something fishy. I would expect all tests to run because I changed the eslint config. Let me do more tests locally.

**Comment by julien51** [comment]
confirmed. I was using an array var in the bash script when it should be a string. pushing a fix.

**Pull request merged**